rules/manager_test.go

```diff
@@ -353,176 +353,182 @@ func sortAlerts(items []*Alert) {
 }
 
 func TestForStateRestore(t *testing.T) {
-	storage := promqltest.LoadedStorage(t, `
+	for _, queryOffset := range []time.Duration{0, time.Minute} {
+		t.Run("queryOffset "+queryOffset.String(), func(t *testing.T) {
```

Comment:
Same here. Not sure what I'm missing 🤔 

```suggestion
		t.Run(fmt.Sprintf("queryOffset %s", queryOffset.String()), func(t *testing.T) {
```